diff -urN aaa/downman-0.0.2/configure.in downman-0.0.2/configure.in
--- aaa/downman-0.0.2/configure.in	2003-06-01 20:23:39.000000000 +0200
+++ downman-0.0.2/configure.in	2003-07-25 23:40:27.000000000 +0200
@@ -40,6 +40,4 @@
 clients/gnome/Makefile
 clients/gnome/monitor/Makefile
 clients/gnome/gdownman/Makefile
-dnl intl/Makefile
-dnl po/Makefile.in
-])
\ Brak znaku nowej linii na koñcu pliku
+])
